server/channels/app/import_functions_test.go

```diff
@@ -900,22 +910,26 @@ func TestImportImportUser(t *testing.T) {
 
 	// Test team and channel memberships
 	teamName := model.NewRandomTeamName()
-	th.App.importTeam(th.Context, &imports.TeamImportData{
+	err = th.App.importTeam(th.Context, &imports.TeamImportData{
```

Comment:
`th.App.importTeam` returns a `*model.AppError`. You can't assign that to an `error ` variable.

Reply:
ah right, got it. thx for the hint.